data-import.qmd

```diff
@@ -335,14 +425,14 @@ There are two alternatives:
     read_rds("students.rds")
     ```
 
-2.  The feather package implements a fast binary file format that can be shared across programming languages:
+2.  The arrow package allows you to read and write parquet files, a fast binary file format that can be shared across programming languages:
```

Comment:
Should add cross-ref to the new binary data chapter?

Reply:
Yeah when it exists 😄